migrations/1685969609_addSectionTeamGallery.ts

```diff
@@ -0,0 +1,75 @@
+import { Client, SimpleSchemaTypes } from "@datocms/cli/lib/cma-client-node";
+
+export default async function (client: Client) {
+  const newFields: Record<string, SimpleSchemaTypes.Field> = {};
+  const newItemTypes: Record<string, SimpleSchemaTypes.ItemType> = {};
+
+  console.log("Create new models/block models");
+
+  console.log(
+    'Create block model "Section Team Gallery" (`section_team_gallery`)'
+  );
+  newItemTypes["1602503"] = await client.itemTypes.create(
+    {
+      name: "Section Team Gallery",
+      api_key: "section_team_gallery",
+      modular_block: true,
+      hint: "https://www.datocms-assets.com/6524/1685609517-section-team-gallery-preview.png",
+      inverse_relationships_enabled: false,
+    },
+    { skip_menu_item_creation: true }
+  );
+
+  console.log("Creating new fields/fieldsets");
+
+  console.log(
+    'Create Multiple links field "Team" (`team`) in block model "Section Team Gallery" (`section_team_gallery`)'
+  );
+  newFields["8299171"] = await client.fields.create(newItemTypes["1602503"], {
+    label: "Team",
+    field_type: "links",
+    api_key: "team",
+    validators: {
+      items_item_type: {
+        on_publish_with_unpublished_references_strategy: "fail",
+        on_reference_unpublish_strategy: "delete_references",
+        on_reference_delete_strategy: "delete_references",
+        item_types: ["38233"],
+      },
+    },
+    appearance: { addons: [], editor: "links_select", parameters: {} },
+  });
+
+  console.log("Update existing fields/fieldsets");
+
+  console.log(
+    'Update Boolean field "Full width" (`full_width`) in block model "Image" (`image`)'
+  );
+  await client.fields.update("159787", { hint: null });
```

Comment:
Is this hint removed by incident?

Reply:
I don't remember removing hints and I can't really tell which field this. Should i run de migration again or fork primary?

Reply:
Running the migration again should end up with exactly the same result. I think you've generated your migration against a sandbox environment that was a bit behind the primary environment and therefore once you generated the migration it thought you removed this hint.

I think it would be best to just remove these lines of code from the migration.